packages/common/http/testing/src/provider.ts

```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {HttpBackend, HttpClientModule} from '@angular/common/http';
+import {HttpBackend} from '@angular/common/http';
 import {NgModule, Provider} from '@angular/core';
```

Comment:
Could you please also remove the `NgModule` symbol import (it's also unused)?
```suggestion
import {Provider} from '@angular/core';
```